service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinTest.java

```diff
@@ -208,7 +200,7 @@ public void getTwinOperationSucceeds(@Mocked DeviceTwinDevice mockedDevice, @Moc
                 mockedHttpRequest.setReadTimeoutMillis(anyInt);
                 times = 1;
                 mockedHttpRequest.setHeaderField(anyString, anyString);
-                times = 5;
+                times = 4;
```

Comment:
why this change?

Reply:
Some of the mocking in the unit test broke when I switched to validating the requestId parameter by calling Tools.isNullOrEmpty, so the header case for adding a request Id header isn't called from this unit test anymore